caesarsCipher/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <title>Caesars Cipher</title>
+
+        <link rel="stylesheet" href="../styles/projectsStyle.css">
+    </head>
+    <body>
+        <div class="container">
+            <h1 class="title">CAESARS CIPHER</h1>
+
+            <div class="container-block">
+                <div>
+                    <form class="form" id="my-form">
+                        <p class="text">Enter your word:<input type="text" maxlength="30" class="input" id="user-input"></p>
```

Comment:
What if the requirement is to support the length up to 1000 symbols?
How can we fit large strings on the screen?

Reply:
fixed